src/types/tmdb.ts

```diff
@@ -318,3 +318,10 @@ export interface ShowData {
         }
     ];
 }
+
+export interface RecommendationData {
```

Comment:
Any reason not to use `interface MovieData` here? Or is this for readability/organization?

Reply:
Updated to using ResultsData